src/CRC.jl

```diff
@@ -35,9 +35,17 @@ import Base.Cartesian: @nexprs
 import Base: ==, isless, print
 using Compat
 
-typealias U Unsigned
-
+const U = Unsigned
 
+immutable UnalignedVector{T<:U}
+    a::Vector{UInt8}
+end
+# unsafe
+@inline Base.getindex{T}(a::UnalignedVector{T}, i) = unsafe_load(Ptr{T}(pointer(a.a)), i)
```

Comment:
(Also because this package still support 0.4, which doesn't support the new bounds check elision yet?)